src/pluto/pluto.py

```diff
@@ -509,6 +509,57 @@ def create_expected(df, v1, v2):
                     st.markdown(f"* in {v2} but not in {v1}:")
                     st.write(in2not1)
 
+
+    def create_outlier(df, v1, v2, condo, mapped):
+        outlier = df.loc[
+            (df.condo == condo)
+            & (df.mapped == mapped)
+            & (df.v == v1),
+            :,
+        ]
+
+        outlier_records = outlier.to_dict("records")
+        v1_outlier = [i["outlier"] for i in outlier_records if i["v"] == v1][0]
+
+        def display_dataframe(v1_outlier, field):
+            records = [i["values"] for i in v1_outlier if i["field"] == field][0]
+            if records:
+                df = pd.DataFrame(records)
+                if field == 'building_area_increase':
+                    df=df.drop(columns=['pair'])
+                df['bbl']=pd.to_numeric(df['bbl'], downcast='integer')
+                return df,df.shape[0]
+            else:
+                return 'This is no outlier.'
+        
+        version_pair = v1 + '-' + v2
+        st.markdown(f'### Table of BBLs with Unreasonable Increase in Building Area {version_pair}')
+        result1=display_dataframe(v1_outlier,'building_area_increase')
+        if type(result1) == str:
```

Comment:
with the above change, this could become a new display_dataframe function like
```
def display_dataframe(v1_outlier, 'building_area_increase'):
  df = filter_records_by_field(v1_outlier, 'building_area_increase')
  if df.empty:
     "There are no outliers."
  else:
    AgGrid(df)
    st.info(f'There are {df.shape[0]} outliers in total.')
``` 
You could move the rest of that info callout into the markdown description above, so it looks like
```
st.markdown(f'### Table of BBLs with Unreasonable Increase in Building Area {version_pair}.
This table displays all BBLs where building area has more than doubled since the previous version')

display_dataframe(v1_outlier, 'building_area_increase')

Reply:
great suggestion